fedmsg_meta_umb/tests/test_tps.py

```diff
@@ -0,0 +1,183 @@
+    lastSuccessfulBuild/artifact/docs/index.html>`_ performs basic package sanity testing of rpms.
+
+    Messages (like the example given here) are published when an **TPS**
+    job is **completed**. (A TPS job performs basic sanity testing for the given brew build).
+    """
+    expected_title = 'tps.completed'
+    expected_link = ('https://rpm-factory-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/'
+                     'test-package-sanity-development/label=ose-slave-tps,provision_arch=x86_64/1915/')
+    expected_subti = 'package sanity testing completed for component: setup-2.8.71-7.el7_4 with brew task_id: 14393825'
+    expected_packages = set(['setup'])
+    expected_icon = 'https://rpm-factory-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/static/0e416130/images/headshot.png'
+    msg = {
+        "username": None,
+        "source_name": "datanommer",
+        "certificate": None,
+        "i": 0,
+        "timestamp": 1509104805.0,
+        "msg_id": "ID:somebroker-43669-1508744086399-2:275325:0:0:1",
+        "crypto": None,
+        "topic": "/topic/VirtualTopic.eng.tps.completed",
+        "headers": {
+            "ci_type": "ci-tps",
+            "content-length": "41",
+            "expires": "0",
+            "brew_task_id": "14393825",
+            "JMS_AMQP_MESSAGE_FORMAT": "0",
+            "JMS_AMQP_NATIVE": "false",
+            "destination": "/topic/VirtualTopic.eng.tps.completed",
+            "component": "setup-2.8.71-7.el7_4",
+            "priority": "4",
+            "message-id": "ID:somebroker-43669-1508744086399-2:275325:0:0:1",
+            "jenkins_build_url": "https://rpm-factory-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/"
+                                 "test-package-sanity-development/label=ose-slave-tps,provision_arch=x86_64/1915/",
+            "timestamp": "0",
+            "JMS_AMQP_FirstAcquirer": "false",
+            "subscription": "/queue/Consumer.client-datanommer.openpaas-prod.VirtualTopic.eng.>"
+        },
+        "signature": None,
+        "source_version": "0.8.2",
+        "msg": {
+            "end_time": "2017-10-27T11:47:02+00:00"
+        }
+    }
+
+
+class TestTPSResults(fedmsg.tests.test_meta.Base):
+    """ `TPS-next <https://rpm-factory-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/test-package-sanity-documentation/
+    lastSuccessfulBuild/artifact/docs/index.html>`_ performs basic package sanity testing of rpms.
+
+    Messages (like the example given here) are published when an **TPS**
+    job is **performed**. (A TPS job performs basic sanity testing for the given brew build).
+    """
+    expected_title = 'tps'
+    expected_link = ('https://rpm-factory-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/'
+                     'test-package-sanity-development/label=ose-slave-tps,provision_arch=x86_64/1915/')
+    expected_subti = 'package sanity testing results for component: setup-2.8.71-7.el7_4 with brew task_id: 14393825'
```

Comment:
A soft recommendation:  put the value of `tps_status` somewhere in the subtitle for this one.

Something like:  `'package sanity testing PASSED for component: setup....'`.  What do you think?  Human readers will be able to see what they want to know without having to click through.